ignite/pkg/goanalysis/goanalysis_test.go

```diff
@@ -2,14 +2,20 @@ package goanalysis_test
 
 import (
 	"errors"
+	"go/ast"
+	"go/parser"
+	"go/token"
 	"os"
 	"path/filepath"
+	"sort"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
 	"github.com/ignite/cli/ignite/pkg/goanalysis"
 	"github.com/ignite/cli/ignite/pkg/xast"
+
+	"golang.org/x/tools/go/ast/astutil"
```

Comment:
This import should be part of a higher level group with `testify`:

```go
"github.com/stretchr/testify/require"
"golang.org/x/tools/go/ast/astutil"
```

Reply:
yes, I saw this one, but every time we run `gofmt`, the command splits the imports this way.